html final/index.html

```diff
@@ -0,0 +1,201 @@
+                                    <li ><a href="">History & Mision</a></li>
+                                    <li><a href="">Library projects</a></li>
+                                    <li><a href="">Library operating hours</a></li>
+                                </ul>
+                            </div>
+                        
+                        </li>
+                        <li class = "about"><a href="">Search collections</a><hr>
+                            <div class="sub-menu1">
+                                <ul>
+                                    <li ><a href="">Library catalog</a></li>
+                                    <li><a href="">eSources</a></li>
+                                    <li><a href="">WorldCat Discovery</a></li>
+                                    <li ><a href="">AUA DIGITAL REPO</a></li>
+                                    <li><a href="">e-Journal finder</a></li>
+                                    <li><a href="">Periodicals</a></li>
+                                </ul>
+                            </div>
+                        
+                        </li>
+                        <li class = "about"><a href="">Help and Guidelines</a><hr>
+                            <div class="sub-menu1">
+                                <ul>
+                                    <li ><a href="">Library Guides</a></li>
+                                    <li><a href="">Quick app for searching databases off campus</a></li>
+                                    <li><a href="">How to book online</a></li>
+                                    <li ><a href="">10 tips to discern fake news</a></li>
+                                    <li><a href="">Library faciclities</a></li>
+                                    <li><a href="">It facilities</a></li>
+                                </ul>
+                            </div>
+                        
+                        </li>
+                        <li class = "about"><a href="">Library services</a><hr>
+                            <div class="sub-menu1">
+                                <ul>
+                                    <li ><a href="">Services for Facilities</a></li>
+                                    <li><a href="">Services for Students</a></li>
+                                    <li><a href="">Services for staff</a></li>
+                                </ul>
+                            </div>
+                        
+                        </li>
+                        <li><a href="">Location and contacts</a><hr></li>
+                    </ul>
+                </div>
+            </div>
+        </header>
+
+       
+            <div class ="middle">
+                <img src="img/01.jpeg" alt="book-photo">
+                <button>Local catalog</button>
+                <button>WorldCat Discovery</button>
+                
+                    <form action="" class = "form1">                        
```

Comment:
action attribute should be non-empty. You can just delete it
```suggestion
                    <form class = "form1">                        
```